NEWS.md

```diff
@@ -4,36 +4,39 @@
 
 **1. The interface for specifying anisotropy operators has changed.**
 
-Anisotropy can now be expressed as a polynomial in spin operators `𝒮[i]`, or as
+Anisotropy can now be expressed as a polynomial in spin operators `𝒮[α]`, or as
 a linear combination of Stevens operators `𝒪[k,q]`. For example,
 ```julia
 a1 = 20*(𝒮[1]^4 + 𝒮[2]^4 + 𝒮[3]^4)
 a2 = 𝒪[4,0] + 5𝒪[4,4]
 ```
 
-These operators can be used as-is for SU(_N_) classical spin dynamics.
+<!-- These operators can be used as-is for SU(_N_) classical spin dynamics.
 Alternatively, in "dipole mode" (the large _S_ classical limit), spin operators
 will be replaced by their expectation values, and each Stevens operator becomes
 a homogeneous polynomial in the expected spin components. For example,
 ```julia
 print_anisotropy_as_classical_spins(𝒪[4,0]) 
 # Output: 3𝒮₁⁴ + 6𝒮₁²𝒮₂² - 24𝒮₁²𝒮₃² + 3𝒮₂⁴ - 24𝒮₂²𝒮₃² + 8𝒮₃⁴
 ```
+-->
```

Comment:
I suggest we remove `print_anisotropy_as_classical_spins()` from the public API, although it is still present in the code. It seems better to refer people to the documentation on Stevens operators.

Reply:
Sounds good to me.